src/devices/shelly.ts

```diff
@@ -1,33 +1,233 @@
+import {Zcl} from "zigbee-herdsman";
 import * as m from "../lib/modernExtend";
 import type {DefinitionWithExtend} from "../lib/types";
+import {assertString} from "../lib/utils";
+
+function addCustomClusterShellyRPC() {
+    return [
+        m.deviceAddCustomCluster("shellyRPCCluster", {
+            ID: 0xfc01,
+            manufacturerCode: Zcl.ManufacturerCode.SHELLY,
+            attributes: {
+                Data: {ID: 0x0000, type: Zcl.DataType.CHAR_STR},
+                TxCtl: {ID: 0x0001, type: Zcl.DataType.UINT32},
+                RxCtl: {ID: 0x0002, type: Zcl.DataType.UINT32},
```

Comment:
```suggestion
                rxCtl: {ID: 0x0002, type: Zcl.DataType.UINT32},
```

(and change the casing of the other too)